qt_dotgraph/package.xml

```diff
@@ -20,6 +20,7 @@
 
   <test_depend>ament_cmake_pytest</test_depend>
   <test_depend>ament_lint_auto</test_depend>
+  <test_depend>ament_lint_common</test_depend>
   <test_depend>ament_flake8</test_depend>
   <test_depend>ament_pep257</test_depend>
   <test_depend>python3-pytest</test_depend>
```

Comment:
I think these three test dependencies aren't needed anymore?

Reply:
Removed

Reply:
The `python3-pytest` test dependency is still listed. What is it needed for?

Reply:
Removed